docs/source/user/installation.rst

```diff
@@ -0,0 +1,111 @@
+.. _numba-cuda-installation:
+
+============
+Installation
+============
+
+Requirements
+============
+
+Supported GPUs
+--------------
+
+Numba supports all NVIDIA GPUs that are supported by the CUDA Toolkit it uses.
+Presently for CUDA 11 this ranges from Compute Capabilities 3.5 to 9.0, and for
+CUDA 12 this ranges from 5.0 to 12.1, depending on the exact installed version.
+
+
+Supported CUDA Toolkits
+-----------------------
+
+Numba-CUDA aims to support all minor versions of the two most recent CUDA
+Toolkit releases. Presently 11 and 12 are supported; CUDA 11.2 is the minimum
+required, because older releases (11.0 and 11.1) have a version of NVVM based on
+a previous and incompatible LLVM version.
+
+For further information about version compatibility between toolkit and driver
+versions, refer to :ref:`minor-version-compatibility`.
+
+
+Installation with a Python package manager
+==========================================
+
+Conda users can install the CUDA Toolkit into a conda environment.
+
+For CUDA 12, ``cuda-nvcc`` and ``cuda-nvrtc`` are required::
+
+    $ conda install -c conda-forge numba-cuda cuda-nvcc cuda-nvrtc "cuda-version>=12.0"
```

Comment:
Should we consider just adding these to the conda recipe?

Reply:
To add, once we depend on `cuda-bindings` in the conda recipe, `cuda-nvrtc` is already a dependency. What is `cuda-nvcc` needed for?

Reply:
I'm stretching my memory a bit here, but I think it might have been the only way to get libdevice installed (I'm not sure if that's still current).

Reply:
`cuda-nvcc` was probably used to provide libnvvm. I need to look it up and check which 12.x did we split it from nvcc-impl.

Reply:
I just remembered - if you want to run the full test suite, you need to have nvcc installed to build the test binaries. I don't know whether this means it makes sense to have it installed for end-users though.